forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -767,6 +767,12 @@ public boolean turnFaceUp(SpellAbility cause) {
 
     public boolean turnFaceUp(boolean manifestPaid, boolean runTriggers, SpellAbility cause) {
         if (isFaceDown()) {
+            // Check replacement effects
+            Map<AbilityKey, Object> repParams = AbilityKey.mapFromAffected(this);
+            List<ReplacementEffect> list = game.getReplacementHandler().getReplacementList(ReplacementType.TurnFaceUp,
```

Comment:
> While Karlov Watchdog is on the battlefield, opponents can't attempt to turn face-down creatures face up either by paying a disguise or morph cost or by paying the mana cost of a cloaked or manifested creature.

Reply:
Ah there it is, thanks

Reply:
well this part shouldn't have disappeared now since it also needs to account for stuff like _Break Open_

Reply:
Just bad rebasing, sorry